### ISSUE TITLE
📐 Theory — 2026-04-06 (22 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#66]
### 🏆 Oral

**1. [Generalized Linear Mode Connectivity for Transformers](https://openreview.net/forum?id=KurYdcCbjv)**
  *Alexander Theus, Alessandro Cabodi, Sotiris Anagnostidis et al.*
  📍 NeurIPS 2025 oral
  > Understanding the geometry of neural network loss landscapes is a central question in deep learning, with implications for generalization and optimization. A striking phenomenon is $\textit{linear ...

**2. [Implicit Regularization for Tubal Tensor Factorizations via Gradient Descent](https://openreview.net/forum?id=2GmXJnyNM4)**
  *Santhosh Karnik, Anna Veselovska, Mark Iwen et al.*
  📍 ICML 2025 oral
  > We provide a rigorous analysis of implicit regularization in an overparametrized tensor factorization problem beyond the lazy training regime. For matrix factorization problems, this phenomenon has...

### ⭐ Spotlight

**3. [Sample-Adaptivity Tradeoff in On-Demand Sampling](https://openreview.net/forum?id=gaHjGx1cMh)**
  *Nika Haghtalab, Omar Montasser, Mingda Qiao*
  📍 NeurIPS 2025
  > We study the tradeoff between sample complexity and round complexity in *on-demand sampling*, where the learning algorithm adaptively samples from $k$ distributions over a limited number of rounds....

**4. [Tight Generalization Bounds for Large-Margin Halfspaces](https://openreview.net/forum?id=wAq0ZLxrGq)**
  *Kasper Green Larsen, Natascha Schalburg*
  📍 NeurIPS 2025
  > We prove the first generalization bound for large-margin halfspaces that is asymptotically tight in the tradeoff between the margin, the fraction of training points with the given margin, the failu...

**5. [Unveiling the Power of Multiple Gossip Steps: A Stability-Based Generalization Analysis in Decentralized Training](https://openreview.net/forum?id=V8dGVO5Xpg)**
  *Qinglun Li, Yingqi Liu, Miao Zhang et al.*
  📍 NeurIPS 2025
  > Decentralized training removes the centralized server, making it a communication-efficient approach that can significantly improve training efficiency, but it often suffers from degraded performanc...

**6. [Hamiltonian Descent Algorithms for Optimization: Accelerated Rates via Randomized Integration Time](https://openreview.net/forum?id=nW6SMcfDq3)**
  *Qiang Fu, Andre Wibisono*
  📍 NeurIPS 2025
  > We study the Hamiltonian flow for optimization (HF-opt), which simulates the Hamiltonian dynamics for some integration time and resets the velocity to $0$ to decrease the objective function; this i...

**7. [The Implicit Bias of Structured State Space  Models Can Be Poisoned With Clean Labels](https://openreview.net/forum?id=3UaItHVjyE)**
  *Yonatan Slutzky, Yotam Alexander, Noam Razin et al.*
  📍 NeurIPS 2025
  > Neural networks are powered by an implicit bias: a tendency of gradient descent to fit training data in a way that generalizes to unseen data. A recent class of neural network models gaining increa...

**8. [How many measurements are enough? Bayesian recovery in inverse problems with general distributions](https://openreview.net/forum?id=IIiRwgkZcm)**
  *Ben Adcock, Nick Huang*
  📍 NeurIPS 2025
  > We study the sample complexity of Bayesian recovery for solving inverse problems with general prior, forward operator and noise distributions. We consider posterior sampling according to an approxi...

**9. [Fast Projection-Free Approach (without Optimization Oracle) for Optimization over Compact Convex Set](https://openreview.net/forum?id=bP5cU0OYSn)**
  *Chenghao Liu, Enming Liang, Minghua Chen*
  📍 NeurIPS 2025
  > Projection-free first-order methods, e.g., the celebrated Frank-Wolfe (FW) algorithms, have emerged as powerful tools for optimization over simple convex sets such as polyhedra, because of their sc...

**10. [Shortcut Features as Top Eigenfunctions of NTK: A Linear Neural Network Case and More](https://openreview.net/forum?id=TYroQXu6X0)**
  *Jinwoo Lim, Suhyun Kim, Soo-Mook Moon*
  📍 NeurIPS 2025
  > One of the chronic problems of deep-learning models is shortcut learning. In a case where the majority of training data are dominated by a certain feature, neural networks prefer to learn such a fe...

**11. [On Agnostic PAC Learning in the Small Error Regime](https://openreview.net/forum?id=6LOgOsIcXe)**
  *Julian Asilis, Mikael Møller Høgsgaard, Grigoris Velegkas*
  📍 NeurIPS 2025
  > Binary classification in the classic PAC model exhibits a curious phenomenon: Empirical Risk Minimization (ERM) learners are suboptimal in the realizable case yet optimal in the agnostic case. Roug...

**12. [Any-stepsize Gradient Descent for Separable Data under Fenchel–Young Losses](https://openreview.net/forum?id=D6aCr4RRdt)**
  *Han Bao, Shinsaku Sakaue, Yuki Takezawa*
  📍 NeurIPS 2025
  > The gradient descent (GD) has been one of the most common optimizer in machine learning. In particular, the loss landscape of a neural network is typically sharpened during the initial phase of tra...

### 📌 Poster

**13. [Tight analyses of first-order methods with error feedback](https://openreview.net/forum?id=hlPk6Hi43e)**
  *Daniel Berg Thomsen, Adrien Taylor, Aymeric Dieuleveut*
  📍 NeurIPS 2025 poster
  > Communication between agents often constitutes a major computational bottleneck in distributed learning. One of the most common mitigation strategies is to compress the information exchanged, there...

**14. [Towards Generalizable 3D Human Pose Estimation via Ensembles on Flat Loss Landscapes](https://openreview.net/forum?id=nZ4mFzCZIx)**
  *Jumin Han, Jun-Hee Kim, Seong-Whan Lee*
  📍 NeurIPS 2025 poster
  > 3D Human Pose Estimation (HPE) is a fundamental task in the computer vision. Generalization in 3D HPE task is crucial due to the need for robustness across diverse environments and datasets. Existi...

**15. [A geometric framework for momentum-based optimizers for low-rank training](https://openreview.net/forum?id=cCefuzQrjK)**
  *Steffen Schotthöfer, Timon Klein, Jonas Kusch*
  📍 NeurIPS 2025 poster
  > Low-rank pre-training and fine-tuning have recently emerged as promising techniques for reducing the computational and storage costs of large neural networks. Training low-rank parameterizations ty...

**16. [Iterative Missing Data Imputation with Model Form Adaptation and Non-Missing Feature Supervision](https://openreview.net/forum?id=L84DdFuvwV)**
  *Hao Wang, zhengnan li, Zhichao Chen et al.*
  📍 NeurIPS 2025 poster
  > Iterative imputation is a prevalent method for missing data imputation, where each feature is imputed iteratively by treating it as a target variable estimated from all other features. However, ite...

**17. [Exploring Landscapes for Better Minima along Valleys](https://openreview.net/forum?id=XxRKqFsvoK)**
  *Tong Zhao, Jiacheng Li, Yuanchang Zhou et al.*
  📍 NeurIPS 2025 poster
  > Finding lower and better-generalizing minima is crucial for deep learning. However, most existing optimizers stop searching the parameter space once they reach a local minimum. Given the complex ge...

**18. [How Compositional Generalization and Creativity Improve as Diffusion Models are Trained](https://openreview.net/forum?id=1OUEnfusEd)**
  *Alessandro Favero, Antonio Sclocchi, Francesco Cagnetta et al.*
  📍 ICML 2025 poster
  > Natural data is often organized as a hierarchical composition of features. How many samples do generative models need in order to learn the composition rules, so as to produce a combinatorially lar...

**19. [Rethinking Benign Overfitting in Two-Layer Neural Networks](https://openreview.net/forum?id=Uc0dTE2Wox)**
  *Ruichen Xu, Kexin Chen*
  📍 ICML 2025 poster
  > Recent theoretical studies (Kou et al., 2023; Cao et al., 2022) revealed a sharp phase transition from benign to harmful overfitting when the
noise-to-feature ratio exceeds a threshold—a situation ...

**20. [The Importance of Being Lazy: Scaling Limits of Continual Learning](https://openreview.net/forum?id=edhBkkYS8R)**
  *Jacopo Graldi, Alessandro Breccia, Giulia Lanzillotta et al.*
  📍 ICML 2025 poster
  > Despite recent efforts, neural networks still struggle to learn in non-stationary environments, and our understanding of catastrophic forgetting (CF) is far from complete.
In this work, we perform ...

**21. [Generalization of noisy SGD in unbounded non-convex settings](https://openreview.net/forum?id=Au9rfI6Fjd)**
  *Leello Tadesse Dadi, Volkan Cevher*
  📍 ICML 2025 poster
  > We study generalization of iterative noisy gradient schemes on smooth non-convex losses. Formally, we establish time-independent information theoretic generalization bounds for Stochastic Gradient ...

**22. [High Probability Bounds for Non-Convex Stochastic Optimization with Momentum](https://openreview.net/forum?id=KirKWFPYJA)**
  *Shaojie Li, Pengwei Tang, Bowei Zhu et al.*
  📍 ICLR 2026 Poster
  > Stochastic gradient descent with momentum (SGDM) is widely used in machine learning, yet high-probability learning bounds for SGDM in non-convex settings remain scarce. In this paper, we provide hi...
